# This is a failing example. It makes sure, if we fail an assertion, that the program actually fails.
# Exception: assertion failed

assert(1 == 0)